_schemaV2.graphql

```diff
@@ -11178,7 +11178,7 @@ type Message implements Node {
   body: String
 
   # Masked emails w/ display name of the recipients in copy.
-  cc: [String]
+  cc: [String!]!
```

Comment:
this doesn't look like accurate, I would think something like `[String]!`, if we use the way it is now we're considering we always have an element `String`, don't we?

Reply:
I had the same thought [here](https://github.com/artsy/metaphysics/pull/4698#discussion_r1081704058).

Reply:
@araujobarret I know it seems a bit counter intuitive but it _is_ accurate.

Consider the difference between:

```ts
const a: string[] = []
const b: (string | null)[] = []
const c: (string | null)[] | null = []
```